docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/src/salesforce_ftp_integration/Module.md

```diff
@@ -108,17 +60,7 @@ FTP_PATH="<Path in the FTP server you added conatats.csv (eg: /home/ftp-user/sfd
 FTP_POLLING_INTERVAL=<FTP listner polling interval (eg: 600000)>
 ```
 
-#### 6. Write the integration
-
-Open the project with VS Code. The integration implementation is written in the `src/salesforce_ftp_integration/main.bal` file.
-
-Here the `ftpServerConnector` service is running on `remoteServer`, which listens to the configured FTP server
-location. When a CSV file is added to the FTP server, the file content will be retrieved and inserted into the
-Salesforce using `sfBulkClient`.
-
-## Testing
-
-First let’s build the module. Navigate to the project root directory and execute the following command.
+Let’s build the module. Navigate to the project root directory and execute the following command.
```

Comment:
Shouldn't we build before we test?

Reply:
These configurations are required to build the module.

Reply:
These configurations are required to build the module.